app/server/tests/integration/odin.itest.ts

```diff
@@ -0,0 +1,17 @@
+import { wodin } from "./integrationTest";
+
+describe("odin endpoints", () => {
+    it("get versions", async () => {
+        const response = await wodin.get("/odin/versions");
+
+        expect(response.status).toBe(200);
+
+        expect(response.body.data).toEqual({
+            dfoptim: "0.0.5",
+            dopri: "0.0.12",
+            odin: "1.3.14",
+            "odin.api": "0.1.6",
+            odinjs: "0.0.14"
+        });
+    });
```

Comment:
> Is there a reason why you're not using the `expect` which is included in `supertest` (as shown in their docs example)? It seems like that's the main advantage of using `supertest` over just axios + jest expect, since it allows the tests to be a bit terser?

The syntax is totally different from our tests code base and its just a matter of choice so I decided to use async/await syntax to avoid confusion when context switching.